hazelcast/src/main/java/com/hazelcast/spi/ExecutionService.java

```diff
@@ -103,6 +104,8 @@
 
     ManagedExecutorService register(String name, int poolSize, int queueCapacity, ExecutorType type);
 
+    ManagedExecutorService register(String name, int poolSize, int queueCapacity, ThreadFactory threadFactory);
```

Comment:
Since you already touched here, can you add javadoc to these methods? Especially this one doesn't take an `ExecutorType` but the above one does.

Reply:
done